harmoni_detectors/harmoni_face_detect/launch/include/dlib.launch

```diff
@@ -0,0 +1,9 @@
+<launch>
+   
+    <arg name="instance_id" default="default"/>
+
+    <param name="name" value="dlib"/>
```

Comment:
I think the param "name" is being removed from most packages, with emphasis on the use of instance_id. Is it still needed here?

Reply:
I wasn't sure what the purpose was, so I left it as is. It seemed to be more relevant as a "type" than a "name" in case someone wanted to know what backend was being used for the topic. Could definitely remove it though.

Reply:
done